ansible/roles/ocp4-workload-pipelines/tasks/workload.yml

```diff
@@ -107,10 +111,12 @@
   until:
   - r_pipeline_controller_deployment.resources | length | int > 0
   - r_pipeline_controller_deployment.resources[0].status.availableReplicas is defined
-  - r_pipeline_controller_deployment.resources[0].status.availableReplicas | int == r_pipeline_controller_deployment.resources[0].spec.replicas | int
+  - >-
+    r_pipeline_controller_deployment.resources[0].status.availableReplicas | int
+    == r_pipeline_controller_deployment.resources[0].spec.replicas | int
```

Comment:
This would be more readable if a task var were used for `r_pipeline_controller_deployment.resources[0]`